seed.js

```diff
@@ -0,0 +1,296 @@
+const Seller = require('./db/index.js').Seller;
+
+//for randomReviewItem and photoInComment
+let counter = 0;
+
+// random name generator
+let getRandomNames = () => {
+  let userNames = [
+    'Agaringt',
+    'Amiramp',
+    'Angurismu',
+    'ArticleClown',
+    'Bathervi',
+    'BelieveBandit',
+    'BestCandy',
+    'Blayborg',
+    'Guitarist',
+    'Hannisab',
+    'Hellogyra',
```

Comment:
Consider moving your arrays containing random data to generate from to a different file and then requiring them. It would make this a little easier to read

Reply:
Will make those changes!